<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarCraftSolarSystemOriginsClass</class>
 <widget class="QWidget" name="StarCraftSolarSystemOriginsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarCraftSolarSystemOrigins</string>
  </property>
  <widget class="QLabel" name="img_jugador2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>320</y>
     <width>281</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/MainMenu/Resources/Tagal_Psycher.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="container_titleJugador2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>130</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-style: outset;
border-width: 1px;
border-color: rgb(0, 230, 77);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/MainMenu/Resources/starcraft_bg.jpg);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_iniciar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>600</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 1px;
border-color: rgb(0, 121, 205);
color: rgb(233, 254, 254);
background-color: rgb(7, 92, 182);</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="img_jugador1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/MainMenu/Resources/warrior.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="container_jugador1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>281</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 2px;
border-color: rgb(0, 0, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="container_jugador2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>130</y>
     <width>291</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 2px;
border-color: rgb(0, 230, 77);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titleJugador2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>140</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 230, 77);</string>
   </property>
   <property name="text">
    <string>Jugador  2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nomJugador2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(233, 254, 254);
border-style: outset;
border-bottom-width: 2px;
border-color: rgb(0, 230, 77);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="container_titleJugador1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-style: outset;
border-width: 1px;
border-color: rgb(0, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nomJugador2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>220</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(233, 254, 254);</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nomJugador1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(233, 254, 254);
border-style: outset;
border-bottom-width: 2px;
border-color: rgb(0, 0, 255);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nomJugador1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(233, 254, 254);</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titleJugador1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Starcraft</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Jugador  1</string>
   </property>
  </widget>
  <zorder>bgImage</zorder>
  <zorder>btn_iniciar</zorder>
  <zorder>img_jugador1</zorder>
  <zorder>container_jugador1</zorder>
  <zorder>container_jugador2</zorder>
  <zorder>txt_nomJugador2</zorder>
  <zorder>container_titleJugador1</zorder>
  <zorder>lbl_nomJugador2</zorder>
  <zorder>txt_nomJugador1</zorder>
  <zorder>lbl_nomJugador1</zorder>
  <zorder>lbl_titleJugador1</zorder>
  <zorder>img_jugador2</zorder>
  <zorder>container_titleJugador2</zorder>
  <zorder>lbl_titleJugador2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="starcraftsolarsystemorigins.qrc"/>
 </resources>
 <connections/>
</ui>
